primap2/_merge.py

```diff
@@ -88,6 +89,13 @@ def generate_log_message(da_error: xr.DataArray, tolerance: float) -> str:
     * convert the rest into a pandas dataframe for nice printing
     """
     scalar_dims = [dim for dim in da_error.dims if len(da_error[dim]) == 1]
+    scalar_dims_format = []
+    for dim in scalar_dims:
+        if dim == "time":
+            single_date = date.fromtimestamp(da_error[dim].item() / 1000000000)
+            scalar_dims_format.append(f"{dim}={single_date.strftime('%Y')}")
```

Comment:
What's happening here?

Reply:
converting the timecode to something human readable (a year). It looks nasty so probably I missed the correct function to use.

Reply:
For now, maybe just add a comment which explains what's going on. I also suspect that it doesn't work if you use another resolution for your datetime, but I don't think it's super important.

Reply:
But there was also a bug and the formatting was never used. Fixed now